feat(curriculum): add registration form workshop to cert (#56724)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f5c3e399ff1a05629964e4.md

---
id: 60f5c3e399ff1a05629964e4
title: Step 6
challengeType: 0
dashedName: step-6
---

# --description--

> As suggested by the title, you are creating a form. So, after the `p` element, insert a `form` with an `action` attribute targeting `https://register-demo.freecodecamp.org`.

Reviewer: ```suggestion
As the title suggests, you are creating a form. After the `p` element, insert a `form` element with an `action` attribute that targets `https://register-demo.freecodecamp.org`.
```
